packages/cli/src/commands/init/init.ts

```diff
@@ -175,13 +176,31 @@ async function installDependencies({
   loader.succeed();
 }
 
+function createTemplateUri(options: Options, version: string): string {
+  const isTypescriptTemplate = options.template?.includes(
+    'react-native-template-typescript',
+  );
+
+  if (isTypescriptTemplate && options.template?.includes('@')) {
+    const typescriptTemplateVersion = options.template.split('@')[1];
+    if (semver.gte(typescriptTemplateVersion, '0.71.0')) {
```

Comment:
Not sure if I get this right. Is it the RN version inside the template? The TS template does not follow RN versioning. You can look up what are the versions here https://github.com/react-native-community/react-native-template-typescript#react-native--template-version

If I'm wrong just ignore this comment.

Reply:
> Not sure if I get this right. Is it the RN version inside the template? The TS template does not follow RN versioning. You can look up what are the versions here https://github.com/react-native-community/react-native-template-typescript#react-native--template-version
> 
> If I'm wrong just ignore this comment.

No, you're 💯 correct.
Actually now I'm tending to just doing the check if there's TS template passed but without any version
If users go with `init --template react-native-template-typescript` once r-n 0.71 is released we will be using `react-native`. But if they pass version with template I would assume that they want lower react native version but with TS so we will allow that? 
wdyt?

Reply:
Yeah, looks like that's the way to go!